mllib/src/main/scala/org/apache/spark/mllib/linalg/Vectors.scala

```diff
@@ -38,8 +38,6 @@ import org.apache.spark.sql.types._
 
 /**
  * Represents a numeric vector, whose index type is Int and value type is Double.
- *
- * Note: Users should not implement this interface.
```

Comment:
That's my point with the change. If uses should not implement this interface, we should offer them alternatives. Do you know any? I'm happy to add it to the scaladoc.